src/components/Dialog/Dialog.stories.jsx

```diff
@@ -47,8 +46,8 @@ export default {
 
 const Template = args => {
   return (
-    <Container className={appStyles.App} fixed>
-      <Grid xs={12} container className={headerStyles.Header} item={true}>
+    <Container className={appStyles.movieListPage} fixed>
+      <Grid xs={12} container className={appStyles.appHeader} item={true}>
```

Comment:
better use shortened item
```suggestion
      <Grid xs={12} container className={appStyles.appHeader} item>
```